docs/transactional-semantics.rst

```diff
@@ -0,0 +1,27 @@
+Transactional Semantics
+=======================
+
+In `"Using
+Tendermint"<./specification/using-tendermint.html#broadcast-api>`__ we
+discussed different API endpoints for sending transactions and
+differences between them.
+
+What we have not yet covered is transactional semantics.
+
+When you send a transaction using one of the available methods, it
+first goes to the mempool. Currently, it does not provide strong
+guarantees like "if the transaction were accepted, it would be
+eventually included in a block (given CheckTx passes)."
```

Comment:
we should mention that if you want the guarantees, you can just use `_commit` (which will either be committed or return an error, unlike _sync), but then you have to deal with higher latency. This is the tradeoff.

Reply:
I assumed people have read "Using tendermint Broadcast API" where we explain the difference. But I guess it's worth repeating. Will do

Reply:
Can we not consolidate this doc with that one ?

Reply:
I thought this one will be more technical (mentioning things like WAL, ...)